build/pion-setup.inc

```diff
@@ -40,9 +40,9 @@ case "$build_os" in
 	# -pthread is implied (automatically set by compiler)
 	CPPFLAGS="$CPPFLAGS -D_REENTRANT"
 	#LDFLAGS="$LDFLAGS"
-	#PION_OPT_FLAGS="-O3 -ggdb -fomit-frame-pointer -fstrict-aliasing -fno-tree-pre -falign-loops -Wall -Wno-strict-aliasing -DNDEBUG"
-	PION_OPT_FLAGS="-O3 -ggdb -fomit-frame-pointer -fstrict-aliasing -Wall -Wno-strict-aliasing -DNDEBUG"
-	PION_DEBUG_FLAGS="-O0 -ggdb -Wall -Wno-strict-aliasing -fkeep-inline-functions"
+	#PION_OPT_FLAGS="-O3 -ggdb -fomit-frame-pointer -fstrict-aliasing -fno-tree-pre -falign-loops -Wall -Wextra -Werror -Wno-strict-aliasing -DNDEBUG"
+	PION_OPT_FLAGS="-O3 -ggdb -fomit-frame-pointer -fstrict-aliasing -Wall -Wextra -Werror -Wno-strict-aliasing -DNDEBUG"
+	PION_DEBUG_FLAGS="-O0 -ggdb -Wall -Wextra -Werror -Wno-strict-aliasing -fkeep-inline-functions"
```

Comment:
Unfortunately in this case, it's in the boost library not pion. I found another case like this, but I was able to work around both using #pragma so I didn't have to remove -Werror